fw/stm32_spi.h

```diff
@@ -135,6 +135,51 @@ class Stm32Spi {
     return result;
   }
 
+  // "no_cs" versions of the SPI write functions
+  // these allow us to control the chip select pin outside of these
+  // functions, for cases where it's used as a framing device between
+  // bytes (like the AMT22)
+  // NOTE that these functions, unlike the regular write functions above,
+  // operate with 8 bit values instead of 16 because they were made for
+  // the AMT22, which needs a >2.5us delay in between byte reads
+  uint8_t write_no_cs(uint8_t value) MOTEUS_CCM_ATTRIBUTE {
+    start_write_no_cs(value);
+    return finish_write_no_cs();
+  }
+
+  void start_write_no_cs(uint8_t value) MOTEUS_CCM_ATTRIBUTE {
+    auto* const spi = spi_.spi.handle.Instance;
+
+    // This doesn't seem to be a whole lot faster than the HAL in
```

Comment:
This comment shouldn't be duplicated from above.  At most it should be a reference to the previous comment and might be fine to omit entirely.